libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginAnalyticsListener.kt

```diff
@@ -65,7 +65,6 @@ interface LoginAnalyticsListener {
     fun trackPickEmailFromHint()
     fun trackShowEmailHints()
     fun emailFormScreenResumed()
-    fun trackEmailSignupConfirmationViewed()
```

Comment:
👋 @renanferrari !

Oh you're right, again nice catch! `Woo` implements this method but as a `TODO("Not yet implemented")`, which means that when they pull the changes from this library the compiler will complain. I'll revert that change on the library level and have the `WordPress` app implement it with a `\\ Do nothing` comment. Wdyt?

Reply:
This is now done here: bbf246401b97edb50bfdd794b82d6fa4425cdad6

Reply:
In this case I'm wondering if it's not better to actually remove this, since we are effectively removing the "signup confirmation" screen for email signups. And as none of the apps are going to use it, it seems right. What do you think?

Reply:
👋 @renanferrari !

Do you mean `if it's better to actually remove this`, without the `not`, I just got a bit confused? 😅 

If this is indeed what you meant, I was thinking along the same lines, but decided that it probably better to not have that removed since on the `Woo` side it is marked as `TODO("Not yet implemented")`. We now know that on the `WordPress` side this is a `\\ Do nothing.`, but I am not sure about `Woo`.

Also `Woo` has an intermediate `LoginNoWPcomAccountFoundFragment` screen as well, which made me think that I shouldn't remove this logic from `Woo` just yet (it might be of used as some point, or planned).

<img src="https://user-images.githubusercontent.com/9729923/107741305-936bfa00-6d15-11eb-9b42-5831b599d1e2.png" width="250" height="500">

However, if you really think that it is indeed better if we just remove this, and have `Woo` update their implementation by removing this as well, that is when they pull the latest changes, then I will confidently do so and I would agree. It is just that I choose the less risky, frictionless path on this instance.

Reply:
👋 @renanferrari !

As per our discussion, I removed this tracking event, you can check this commit: 5252eda1c835abb4d51eafed864db6dfe6fb20df

Reply:
👋 @anitaa1990 !

Could you also please verify that this is okay on your end?

Many thanks in advance! 🙏